src/custom-surf/components/cim/ImpactedObjects.tsx

```diff
@@ -248,13 +248,7 @@ const ImpactedObjects = ({ ticket, updateable, withSubscriptions }: IProps) => {
 
     // Hide subscription and other columns when not needed
     if (!withSubscriptions) {
-        columns = columns.filter(
-            (c) =>
-                c.field !== "subscription" &&
-                c.field !== "type" &&
-                c.field !== "impact_override" &&
-                c.field !== "edit_impact_override"
-        );
+        columns = columns.filter((c) => c.field !== "subscription" && c.field !== "type");
```

Comment:
or with a bit of param destructuring:

`columns = columns.filter(({field}) -> field !== "subscription" && field !== "type")`

but unless you do complex things I guess the original code is fine as well

Reply:
```suggestion
        columns = columns.filter(({ field }) => field !== "subscription" && field !== "type");
```